CODEOWNERS

```diff
@@ -5,4 +5,5 @@
 # https://git-scm.com/docs/gitignore#_pattern_format
 
 # Default
-*				@gdbelvin
+*				@google/keytransparency
+core/crypto			@gdbelvin @thaidn @benbenoy
```

Comment:
Are the 2 additional folk here aware that this is incoming?

Reply:
I'll hold off on this until I hear from them.  Thanks for that suggestion.